src/app/model/services/beol.service.ts

```diff
@@ -11,8 +12,8 @@ export class BeolService {
 
         const bookTemplate = `
     PREFIX knora-api: <http://api.knora.org/ontology/knora-api/simple/v2#>
-    PREFIX biblio: <http://0.0.0.0:3333/ontology/biblio/simple/v2#>
-    PREFIX beol: <http://0.0.0.0:3333/ontology/0801/beol/simple/v2#>
+    PREFIX biblio: <${environment.api}/ontology/biblio/simple/v2#>
```

Comment:
This does not work if you are on localhost: instead of `http://0.0.0.0:3333` it says `http://localhost:3333` which is rejected by the Knora API: 

> {"error":"org.knora.webapi.DataConversionException: IRI http://localhost:3333/ontology/0801/beol/simple/v2#letterHasTranslation is not a Knora entity IRI, so it cannot be a link property IRI"}

Reply:
should work after the latest commit.

Reply:
It works!

Should we systematically replace `localhost` entries with `0.0.0.0`?

Reply:
We could, but it shouldn't make any difference. It's only important for the API server since there it is checked.